.github/dependabot.yml

```diff
@@ -2,10 +2,39 @@
 # See https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/keeping-your-actions-up-to-date-with-dependabot
 
 version: 2
+enable-beta-ecosystems: true
 updates:
 
 - package-ecosystem: "github-actions"
   directory: "/"
   schedule:
-    # Check for updates to GitHub Actions every weekday
     interval: "monthly"
+
+- package-ecosystem: "pub"
+  directory: "/"
+  schedule:
+    interval: "weekly"
```

Comment:
monthly? Weekly is pretty...frequent

Reply:
I run `flutter/codelabs` and `flutter/samples` daily just to spread out the notification emails. When it was weekly I used to lose a day to just dealing with all the notifications.

Reply:
🤷